src/core/config/type-cast/index.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * @prettier
+ */
+import has from "lodash/has"
+import get from "lodash/get"
+import set from "lodash/set"
+
+import typeCasters from "./type-casters"
+
+const typeCast = (options) => {
+  return Object.entries(typeCasters).reduce(
+    (acc, [option, typeCaster]) => {
+      if (has(acc, option)) {
+        acc = set(acc, option, typeCaster(get(acc, option)))
```

Comment:
We have there operations in one line here. Let's break it down so that it's a bit more readable.

```suggestion
        const uncasted = get(acc, optionPath)
        const casted = typeCaster(uncasted)
        acc = set(acc, option, casted)
```